runner/diff.go

```diff
@@ -63,3 +69,45 @@ func diffRunner(ctx context.Context, test *model.Test, sub model.KeyedSubmission
 	// TODO(flowlo): Save result.
 	return nil
 }
+func IODiffRun(ctx context.Context, params map[string]string, sub model.KeyedSubmission) error {
+	return errors.New("IO testing not yet available.")
+}
+
+func OutMatchDiffRun(ctx context.Context, params map[string]string, sub model.KeyedSubmission) (testResult model.DiffTestResult, err error) {
+	var str model.SimpleTestResult
+	str, err = Simple(ctx, sub)
+	if err != nil {
+		return
+	}
+	testResult = model.DiffTestResult{
+		SimpleTestResult: str,
+	}
+
+	var rc io.ReadCloser
+	rc, err = storage.NewReader(ctx, params["bucket"], params["tests"])
```

Comment:
hmm..yes..i forgot about that. I`ll create a new runner(ioRunner) because we already have the type there so, we kind of need to split them into 2 different runners. We need to pass the input values to the `params. path["tests"] = "ni:4;fizzbuzz10^4;fi:fizzin;fizzbuzzxxx"`. ni - normal input, fi - file input .`strings.Split(param, ";")` of even numbers is the input and the others the output test. WDYT?